dynamic-plugins/wrappers/backstage-plugin-catalog-backend-module-ldap-dynamic/package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "backstage-plugin-catalog-backend-module-ldap",
+  "version": "0.7.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": true,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin-module",
+    "supported-versions": "1.27.7"
```

can you update this to 1.29.2?

---

Please use suggestions. 

```suggestion
    "supported-versions": "1.29.2"
```